src/Symfony/Component/Notifier/Bridge/SpotHit/phpunit.xml.dist

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you please copy this file from an existing bridge to keep the same formatting? Thanks